components/lock.rst

```diff
@@ -150,6 +150,53 @@ This component also provides two useful methods related to expiring locks:
 ``getExpiringDate()`` (which returns ``null`` or a ``\DateTimeImmutable``
 object) and ``isExpired()`` (which returns a boolean).
 
+The Owner of The Lock
+---------------------
+
+Locks that are acquired for the first time are owned by the ``Lock`` instance[1]_ that acquired
+it. If you need to check whether the current ``Lock`` instance is (still) the owner of
+a lock, you can use the ``isAcquired()`` method::
+
+    if ($lock->isAcquired()) {
+        // We (still) own the lock
+    }
+
+.. note::
+
+    [1] Technically, the true owners of the lock are the ones that share the same instance of ``Key``,
```

Comment:
```suggestion
.. [1] Technically, the true owners of the lock are the ones that share the same instance of ``Key``,
```
should do the trick:
```rst
.. [1] Technically, the true owners of the lock are the ones that share the same instance of ``Key``,
         not ``Lock``. But from a user perspective, ``Key`` is internal and you will likely only be working
         with the ``Lock`` instance so it's easier to think of the ``Lock`` instance as being the one that
         is the owner of the lock.
```

Reply:
Another idea is to move the footnote to the bottom of this section, WDYT?

Reply:
I don't know, is there any rule? 😄

Reply:
No there isn’t, but it’s a technical detail and breaks the reading flow a bit

Reply:
Right, like so then?